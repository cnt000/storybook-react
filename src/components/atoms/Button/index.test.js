import React from 'react'
import Button from './Button'
import renderer from 'react-test-renderer'

it('renders correctly a Button type size', () => {
  const tree = renderer.create(
    <Button type={'size'} onClick={alert('clicked')}>{'42'}</Button>
  ).toJSON();
  expect(tree).toMatchSnapshot();
})