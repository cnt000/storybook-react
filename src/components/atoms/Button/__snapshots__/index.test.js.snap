// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly a Button type size 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "border": "3px solid orange",
      "borderRadius": 3,
      "cursor": "pointer",
      "fontSize": 15,
      "margin": 10,
      "padding": "3px 10px",
    }
  }
>
  42
</button>
`;
